fontspector-py/tests/test_checks_opentype_name.py

```diff
@@ -65,7 +65,7 @@ def test_check_name_no_copyright_on_description():
     )
 
 
-def test_check_monospace():
+def NOT_PORTED_YET_test_check_monospace():
```

Comment:
I don't like these; they basically hide the tests completely. Can we make them `xfail` or `skip` marks?

Reply:
Good idea! I'll do it.

Reply:
done!